ServerlessLibraryAPI/ClientApp/src/components/Contribute/ContributionsList.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Label } from "office-ui-fabric-react";
+import ItemList from "../../components/ItemList/ItemList";
+
+class ContributionsList extends Component {
+  filteredSamples() {
+    let { samples, user } = this.props;
+    let { userName } = user || "__nouser__";
```

Comment:
why?

Reply:
I am giving a dummy user name in case username is undefined (hopefully no sample will match __nouser__ !)